liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -1173,15 +1175,22 @@ protected void doExecute() throws Exception {
                 if (isEar) {
                     runMojo("org.apache.maven.plugins", "maven-ear-plugin", "generate-application-xml");
                     runMojo("org.apache.maven.plugins", "maven-resources-plugin", "resources");
-
                     installEmptyEarIfNotFound(project);
                 } else if (project.getPackaging().equals("pom")) {
                     log.debug("Skipping compile/resources on module with pom packaging type");
                 } else {
                     purgeLocalRepositoryArtifact();
```

Comment:
Why do we still need to purge the local repository?

Reply:
Whoops... bad rebase... I fixed it.   Let me test again though , when I get a chance, to make sure I didn't mess this up.

Reply:
Maybe I could take this to the next step, which I forgot to mention and make sure the watcher compile fails as well.

Reply:
By make sure the watcher compile fails as well, do you mean making sure the watcher compile has `target/classes` in its artifacts list to resolve against?

Reply:
That's right.  I updated the PR for this just now.

Reply:
Thx for taking a quick look.  Sorry for the formatting changes that I see snuck in here.